app/src/main/java/com/github/se/wanderpals/ui/screens/overview/OverviewContent.kt

```diff
@@ -64,7 +65,11 @@ fun OverviewContent(
                   .width(260.dp)
                   .height(55.dp)
                   .testTag("noTripForUserText"),
-          text = "Looks like you have no travel plan yet. ",
+          text =
+              when (SessionManager.getIsNetworkAvailable()) {
+                true -> "Looks like you have no travel plan yet. "
+                false -> "No internet connection."
```

Comment:
Maybe we can use: It seems you're not connected to the Internet

Reply:
Will do, thx!